From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: protect struct cstate
Patch-mainline: never, kabi
References: kabi

In networking-stable-18_04_13, commit
3f01ddb962dc506916c243f9524e8bef97119b77 (slip: Check if rstate is
initialized before uncompressing) added one bool to struct cstate. It
made the kABI checker complaining.

Given the bool fits in a hole after cs_this, just hide the change from
the kABI checker.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 include/net/slhc_vj.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/net/slhc_vj.h
+++ b/include/net/slhc_vj.h
@@ -127,7 +127,9 @@ typedef __u32 int32;
  */
 struct cstate {
 	byte_t	cs_this;	/* connection id number (xmit) */
+#ifndef __GENKSYMS__
 	bool	initialized;	/* true if initialized */
+#endif
 	struct cstate *next;	/* next in ring (xmit) */
 	struct iphdr cs_ip;	/* ip/tcp hdr from most recent packet */
 	struct tcphdr cs_tcp;
